contracts/modules/royalty/policies/IpRoyaltyVault.sol

```diff
@@ -216,6 +226,9 @@ contract IpRoyaltyVault is IIpRoyaltyVault, ERC20SnapshotUpgradeable, Reentrancy
     /// @param token The revenue token to claim
     /// @return The amount of revenue token claimable
     function _claimableRevenue(address account, uint256 snapshotId, address token) internal view returns (uint256) {
+        // if the ip is tagged, then the unclaimed royalties are lost
+        if (DISPUTE_MODULE.isIpTagged(ipId)) return 0;
```

Comment:
Return 0 or should we revert with error?

Reply:
Would suggest return 0 for better composibility - view functions ideally wouldn't revert (also apps don't have to be aware whether a dispute has happened which could make some of their functions revert )